core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1048,7 +1048,7 @@ public static function redirect($strLocation, $intStatus=303)
 		$strLocation = str_replace('&amp;', '&', $strLocation);
 
 		// Make the location an absolute URL
-		if (!preg_match('@^https?://@i', $strLocation))
+		if (!preg_match('@^(https?://|/)@i', $strLocation))
 		{
 			if ($strLocation[0] == '/')
```

Comment:
This seems to be wrong?
`$strLocation[0] == '/'` can never be true because of the `!preg_match()` now.